pkg/platform/local/test/platform_test.go

```diff
@@ -312,6 +312,22 @@ func (suite *TestSuite) TestDeployFunctionDisabledDefaultHttpTrigger() {
 		})
 }
 
+func (suite *TestSuite) TestDeployFunctionWithVeryLongSourceCode() {
+	createFunctionOptions := suite.getDeployOptions("very-long-func")
+	createFunctionOptions.FunctionConfig.Meta.Namespace = suite.namespace
+	createFunctionOptions.FunctionConfig.Spec.Build.FunctionSourceCode = strings.Repeat("t", 200000)
```

Comment:
doesn't it fail if there is no handler?

Reply:
This test will be deleted, I'm working on another one now